app/models/developer.rb

```diff
@@ -22,6 +22,11 @@ class Developer < ApplicationRecord
   validates :cover_image, content_type: ["image/png", "image/jpg", "image/jpeg", "image/gif"],
     max_file_size: 10.megabytes
 
+  validates_numericality_of :preferred_min_hourly_rate, less_than: ->(developer) { developer.preferred_max_hourly_rate }, if: -> { preferred_min_hourly_rate.present? }
```

Comment:
If you add a min but no max hourly or salary an exception is raised.

```
web    | TypeError (can't convert nil into Float):
web    |
web    | app/controllers/developers_controller.rb:38:in `update'
```

I _think_ this is the line, but not 100% sure.